libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -558,7 +558,10 @@ class AP_AHRS
 
     // return current vibration vector for primary IMU
     Vector3f get_vibration(void) const;
-    
+
+    // set and save the ALT_M_NSE parameter value
```

Comment:
This one's making an unwarranted assumption.

Reply:
Txs again for the comments.  I thought as well about giving it a less specific comment but I think there is a good chance that Sub shouldn't be adjusting the EKF's noise parameters this way and it's best to expose this.

Reply:
I think @peterbarker  means that we shouldn't speculate about the ALT_M_NSE parameter here in AHRS, and rather say that we are changing the alt noise setting in the EKF.

Reply:
ok, txs, i've replaced the comment